bundles/org.eclipse.swt/META-INF/p2.inf

```diff
@@ -44,3 +44,11 @@ requires.9.namespace = org.eclipse.equinox.p2.iu
 requires.9.name = org.eclipse.swt.gtk.linux.riscv64
 requires.9.range = [$version$,$version$]
 requires.9.filter = (&(osgi.os=linux)(osgi.ws=gtk)(osgi.arch=riscv64)(!(org.eclipse.swt.buildtime=true)))
+
+# Ensure that 4.36 is not installed with the 0.0.1,2024.4.0.202412101111 version of darkest dark devstyle.
+# https://github.com/eclipse-packaging/packages/issues/307
+requires.100.namespace=org.eclipse.equinox.p2.iu
+requires.100.name=com.genuitec.eclipse.theming.core.feature.feature.group
+requires.100.range = [0.0.1,2024.4.0.202412101111]
```

Comment:
Do we know that they don't auto push things with new qualifier? IMO it's best this to become [0.0.1, 2024.4.1).

Reply:
I really think folks are missing the point and are now split hairs.  We are excluding the broken version and are assuming that any new version will not be broken because that's what folks have promised.